### ISSUE TITLE
mp4 & m4v - Project-Valerie

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Rikitik/project-valerie#266]
```
GERMAN:

Hallo liebe "Project-Valerie" Hersteller und libe "Project-Valerie" Freunde

Zuerst mochte ich mich bei allen bedanken die es ermöglicht haben das wir so 
einen tollen MediaCenter auf unseren Dreambox haben.

Mein Problem:

Grundsätzlich kann meine DreamBox unter dem Original Betriebssystem und  unter 
dem Gemini die Video- Formate: "mp4 & m4v" erkennen, lesen und wiedergeben. 
Meine Frage deshalb: 
Warum erkennt (liest) "Project-Valerie" die Formate nicht obwohl das 
Betriebssystem & Gemini  der DreamBox diese Formate erkennen und wiedergeben 
tut?

Für eine positive Antwort und eine Positive Lösung meines Problems danke ich 
allen im Voraus.
-----------------------------------------------------------------------
ENGLISH:

Hello dear, "Valerie Project" producer and liberal "Valerie Project" Friends

At first I liked me to thank everyone who made it that we have such a great 
media center on our Dreambox.

My problem:

Basically, my DreamBox under the original operating system and the Gemini video 
formats: recognize "mp4 & m4v", read and play.
My question therefore:
Why does (reads) "Valerie Project" does not even recognize the formats of the 
operating system and the Gemini DreamBox these formats and play?

For a positive answer and a positive solution to my problem, I thank everybody 
in advance.

```

Original issue reported on code.google.com by `bhboy...@gmail.com` on 12 Aug 2011 at 11:46
